_topic_map.yml

```diff
@@ -242,10 +242,12 @@ Topics:
   Dir: installing_vsphere
   Topics:
   - Name: Installing a cluster on vSphere
+    File: installing-vsphere-installer-provisioned
```

Comment:
vsphere IPI also support customized installation like other platform, including each type of nodes platform, folder, networking, so suggest to add one topic of "Installing a cluster on vSphere with customization"

Reply:
@patrickdillon 
1. Do you have an example for the allowed scope of customization for vSphere IPI?
2. If I revive https://github.com/openshift/openshift-docs/pull/15348, is the same cluster removal process valid for UPI and IPI, or do you destroy the cluster like with other IPI processes?

Reply:
1. Here is an example of install-config.yaml that I used to test customized installation. The platform both under compute and master sections are default value if user directly install ocp in interactive mode through launching command "openshift-install create cluster".

apiVersion: v1
baseDomain: qe.devcluster.openshift.com
compute:
- hyperthreading: Enabled
  name: worker
  replicas: 3
  platform:
    vsphere:
      cpus: 2
      coresPerSocket: 2
      memoryMB: 8196
      osDisk:
        diskSizeGB: 120
controlPlane:
  hyperthreading: Enabled
  name: master
  replicas: 3
  platform:
    vsphere:
      cpus: 4
      coresPerSocket: 2
      memoryMB: 16384
      osDisk:
        diskSizeGB: 120
metadata:
  name: jimaipi
platform:
  vsphere:
    datacenter: dc1
    network: VM Network
    defaultDatastore: nvme-ds1
    password: <password>
    username: <user_name>
    vCenter: vcsa-qe.vmware.devcluster.openshift.com
    folder: /dc1/jima/jimaipi
    cluster: devel
    apiVIP: <api vip>
    ingressVIP: <ingress vip>
fips: false 
pullSecret: ""
sshKey:

Reply:
Item2 about uninstalling cluster,  destroying cluster like other platform ipi procedures, and this procedure is only used to ipi.